witness/golang/omniwitness/usbarmory/storage/internal.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2022 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build usbarmory
+// +build usbarmory
+
+// Package storage provides support for using the SD/eMMC storage provided
+// by the USB Armory.
```

Comment:
Perhaps some comment about this being extremely raw access, and it being up to the code to make sure it doesn't delete itself if it is installed on the device?

Reply:
done